bin/sqlline-thin.py

```diff
@@ -18,64 +18,58 @@
 # limitations under the License.
 #
 ############################################################################
-
+from __future__ import print_function
 import os
+import phoenix_utils
 import subprocess
 import sys
-import phoenix_utils
-import atexit
 import urlparse
 
-global childProc
-childProc = None
-def kill_child():
-    if childProc is not None:
-        childProc.terminate()
-        childProc.kill()
-        if os.name != 'nt':
-            os.system("reset")
-atexit.register(kill_child)
```

Comment:
I take half of that back. `childProc` is being used in `sqlline.py`, not in this script. I'll see if I can take care of zombie child processes in a cleaner way in both places.